js-functions-arrays/task2.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Створіть функцію яка приймає один параметр value. 
+ * Усередині функції виведіть значення value. 
+ * Після цього, рекурсивно викличте функцію зі
+ *  збільшеним значенням value(наприклад, value + 1).
+ *  Фунція повинна викликатися, доки value не стане менше
+ *  або рівне 20. Викличте вашу функцію з аргументом,
+ *  наприклад, 2.
+ */
+
+function testRecursiveFunction(value) {
+    console.log(value);
+    if (value < 20) {
```

Comment:
"доки value не стане менше,  або рівне 20"
```suggestion
    if (value <= 20) {
```

missed operator.